utils/vara-ui/src/components/select/select.module.scss

```diff
@@ -138,3 +166,72 @@
 .block {
   width: 100%;
 }
+
+:global(.dark-theme) {
```

Comment:
Seems like we need styles for options in dark-theme
<img width="221" alt="image" src="https://github.com/user-attachments/assets/60be3fe8-3ba1-46b0-b1ef-514207ca8de9">